src/layers.jl

```diff
@@ -127,18 +189,32 @@ end
 @functor Patching
 
 """
-    mlpblock(planes, expansion_factor = 4, dropout = 0., dense = Dense)
+    PosEmbedding{T}
 
-Feedforward block used in many vision transformer-like models.
+Positional embedding layer used by many vision transformer-like models. Instantiated with an 
+embedding vector which is a learnable parameter.
+"""
+struct PosEmbedding{T}
+  embedding_vector::T
+end
+
+(p::PosEmbedding)(x) = x .+ p.embedding_vector[:, 1:size(x)[2], :]
```

Comment:
This should be the correct size using the constructor above.

```suggestion
(p::PosEmbedding)(x) = x .+ p.embedding_vector
```

Reply:
Should this constrain the type of `x`, if only as documentation?